### ISSUE TITLE
Inifitite loading terrain - 210.6.5.1 - 1.21

### ISSUE THREAD

**Issue by DrKasango** [mrbuilder1961/WheresMyChatHistory#200]
Infinite loading terrain each time connecting to a server.
Worked just fine for several days after switching to 1.21, the issue started coming up last several days. Sometimes loads after several minute waiting, but mostly resulting in game not responding or even crashing.

Was experiencing the issue even with all the other mods removed (so it does not seem to be a mod conflict).
Issue disappeared only after removing the chatpatches mod.

Crash report:
https://mclo.gs/Dm1Wtka

Please let me know what else is needed to investigate this specific case if makes sense at all.

**Comment by mrbuilder1961** [comment]
thanks, i was going to say this is a duplicate of #191, however the log makes it seem like its #180. either way this is a duplicate issue but i appreciate your report. the most recent comments under #180 have some instructions to use a new build to see if the issue is fixed, i would start there.

**Comment by mrbuilder1961** [comment]
based on the log and your description, i take it that the crash is issue #180 and the long loading times are from #191, although they appear closely related. if you continue to experience these issues, report further information there please! thanks again
